test/integration/targets/ecs_cluster/playbooks/roles/ecs_cluster/tasks/main.yml

```diff
@@ -541,6 +541,23 @@
     # ============================================================
     # Begin tests for Fargate
 
+    - name: ensure AmazonECSTaskExecutionRolePolicy exists
+      iam_role:
+        name: ecsTaskExecutionRole
+        assume_role_policy_document: "{{ lookup('file','ecs-trust-policy.json') }}"
+        description: "Allows ECS containers to make calls to ECR"
+        state: present
+        create_instance_profile: no
+        managed_policy:
+        - AmazonEC2ContainerServiceRole
+        <<: *aws_connection_info
+
+    - name: gather facts for the ecsTaskExecutionRole
```

Comment:
Should be able to access this from `register`ing the results of the previous task

Reply:
I registered the output from iam_role task and reran the tests. The tests passed.